tbx/static_src/sass/components/_featured-services.scss

```diff
@@ -47,9 +47,18 @@
     }
 
     &__heading {
+        @include font-size(size-two-b);
         color: var(--color--heading);
         margin-bottom: $spacer-mini;
-        line-height: 1;
+        word-wrap: break-word;
+        // stylelint-disable-next-line declaration-no-important
+        line-height: 1 !important;
```

Comment:
Rather than using `!important` here, you could just set it on the `--small` variant as well.

Reply:
I actually tried setting the line height on the `--small` variant too initially, but for some reason it still wasn’t being applied. I'm not entirely sure why it’s not taking effect.

Reply:
I'll try this too.

Reply:
Ok I see what the issue is. The [font-size mixin](https://github.com/torchbox/torchbox.com/blob/main/tbx/static_src/sass/config/_mixins.scss#L102-L106) sets the line height at the large media query, whereas we are overriding it at the base size and then at the x-large size.

This seems to be the best solution to avoid the `!important`:

```
&--small {
    @include media-query(large) {
        line-height: 1;
    }

    @include media-query(x-large) {
        @include font-size(size-three);
        line-height: 1;
    }
}

```

Reply:
Interesting, it seems the line-height is being applied now that I've adding the heading classes on the `h3` itself rather than adding the `--small` variant.

Reply:
Looking at what you've done it seems like you're using the existing `heading--three` class instead of creating a new class for `featured-services__heading--small`. If that works OK it's probably a cleaner solution.